packages/rainbowkit/src/components/Avatar/Avatar.tsx

```diff
@@ -53,13 +53,17 @@ export function Avatar({ address, imageUrl, loading, size }: AvatarProps) {
             borderRadius="full"
             height="full"
             position="absolute"
-            style={{ backgroundImage: `url(${imageUrl})` }}
+            style={{
+              backgroundImage: `url(${imageUrl})`,
+              backgroundPosition: 'center',
+            }}
             width="full"
           />
-        ) : null}
-        {emoji}
+        ) : (
+          emoji
+        )}
       </Box>
-      {typeof loading === 'boolean' && (
+      {typeof loading === 'boolean' && loading && (
```

Comment:
@markdalgleish i assume we also want this check for `loading === true`, unless i'm missing something in this check -- the spinner was getting rendered always but had 0 opacity